### PR TITLE
Userstory74 Student Readonly information

### PR DESCRIPTION
**Pull request by 2011sandeepraj**
Testing Examples included.

### CODE REVIEW

File: sources/packages/web/public/index.html [hunk 1]
```diff
@@ -4,14 +4,10 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
```
Comment (andrewsignori-aot) [inline sources/packages/web/public/index.html:7]: Any particular reason to remove these lines now? Would it bring back the cache issue that we faced a few days ago?

File: sources/packages/web/public/index.html [hunk 2]
```diff
@@ -4,14 +4,10 @@
...
-    <meta http-equiv="Pragma" content="no-cache" />
-    <meta http-equiv="Expires" content="0" />
-
     <link rel="icon" href="<%= BASE_URL %>favicon.ico">
     <title><%= htmlWebpackPlugin.options.title %></title>
   </head>
-  <body style="margin: 0px;">
```
Comment (andrewsignori-aot) [inline sources/packages/web/public/index.html:14]: Does it affect the current look of the app?

**Review by andrewsignori-aot — COMMENTED** [review]
Overall changes look good.
If the file sources/packages/web/src/assets/logo.png was added only for the sample page, please consider removing it.

File: sources/packages/web/public/index.html (hunk 1 above)
Comment (2011sandeepraj) [reply sources/packages/web/public/index.html:7]: Got the line back

**Pull request merged**